src/Raven.Client/ServerWide/Commands/GetNodeInfoCommand.cs

```diff
@@ -22,12 +23,34 @@ public class NodeInfo
         public RachisState CurrentState;
         public bool HasFixedPort;
         public int ServerSchemaVersion;
+
+        public DynamicJsonValue ToJson()
+        {
+            return new DynamicJsonValue
+            {
+                [nameof(NodeTag)] = NodeTag,
+                [nameof(TopologyId)] = TopologyId,
+                [nameof(Certificate)] = Certificate,
+                [nameof(ClusterStatus)] = ClusterStatus,
+                [nameof(NumberOfCores)] = NumberOfCores,
+                [nameof(InstalledMemoryInGb)] = InstalledMemoryInGb,
+                [nameof(UsableMemoryInGb)] = UsableMemoryInGb,
+                [nameof(BuildInfo)] = BuildInfo,
+                [nameof(OsInfo)] = OsInfo,
+                [nameof(ServerId)] = ServerId.ToString(),
+                [nameof(CurrentState)] = CurrentState,
+                [nameof(HasFixedPort)] = HasFixedPort,
+                [nameof(ServerSchemaVersion)] = ServerSchemaVersion,
+            };
+        }
     }
 
     public class GetNodeInfoCommand : RavenCommand<NodeInfo>
     {
         public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, ServerNode node, out string url)
         {
+            Timeout = TimeSpan.FromSeconds(15);
```

Comment:
Should be moved to `ctor` at least. Also do we want that for every `GetNodeInfoCommand` or they should use default timeout and only the one created in licensing should have 15sec?

Reply:
done